cmd/antrea-agent/options_windows.go

```diff
@@ -41,10 +40,6 @@ func (o *Options) checkUnsupportedFeatures() error {
 	if o.config.OVSDatapathType != string(ovsconfig.OVSDatapathSystem) {
 		unsupported = append(unsupported, "OVSDatapathType: "+o.config.OVSDatapathType)
 	}
-	_, encapMode := config.GetTrafficEncapModeFromStr(o.config.TrafficEncapMode)
-	if encapMode != config.TrafficEncapModeEncap {
```

Comment:
But we should still check networkPolicyOnly as not supported?

Reply:
Updated.